drivers/mrf24j40/include/mrf24j40_registers.h

```diff
@@ -268,6 +268,14 @@ extern "C" {
 /** @} */
 
 /**
+ * @name    Shift offsets for TXSTAT register (0x24)
+ * @{
+ */
+#define MRF24J40_TXSTAT_MAX_FRAME_RETRIES_SHIFT 6
```

Comment:
please put in parentheses and make explicitly unsigned, i.e `(6U)` - might not be necessary but would be more consistent with (majority) of RIOT code base.

Reply:
Ack, fixed